cinema_service/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("api/cinema/", include("cinema.urls", namespace="cinema"))
```

Comment:
Add `debug_toolbar` please - it will be easier for students to fix all issues connected with SQL requests

Reply:
Done